helpers/sendEmail/emailTemplates.js

```diff
@@ -9,7 +9,7 @@ const emailTemplates = {
     subject: 'Email Verification',
     html: `<h1 style="color: #444; margin-left: 20px;">Welcome to Author's Haven</h1>
 <p style="color: #555; margin-left: 20px; font-size: 14px">Thank you for signing to the Author's Haven. Please click on the button below to activate your account.</p><br>
-<a style="background-color: #61a46e; padding: 12px 15px 12px 15px; color: #eee; font-size: 16px; text-decoration: none; margin-left: 20px; cursor: pointer;" href="http://127.0.0.1:3000/api/users/verify/$">Activate account</a>`
+<a style="background-color: #61a46e; padding: 12px 15px 12px 15px; color: #eee; font-size: 16px; text-decoration: none; margin-left: 20px; cursor: pointer;" href="http://localhost:3000/api/users/verify/$">Activate account</a>`
```

Comment:
It is not good practice to hard code the URL host because the host will keep changing. For example on staging the user will not be able to access localhost. This resource should guide on how to make the host dynamic https://www.w3schools.com/js/js_window_location.asp